client/db/bolt/db.go

```diff
@@ -1131,13 +1127,8 @@ func (db *BoltDB) UpdateOrderMetaData(oid order.OrderID, md *db.OrderMetaData) e
 			put(changeKey, md.ChangeCoin).
 			put(linkedKey, linkedB).
 			put(swapFeesKey, uint64Bytes(md.SwapFeesPaid)).
-			put(maxFeeRateKey, uint64Bytes(md.MaxFeeRate)).
 			put(redemptionFeesKey, uint64Bytes(md.RedemptionFeesPaid)).
-			put(fromVersionKey, uint32Bytes(md.FromVersion)).
-			put(toVersionKey, uint32Bytes(md.ToVersion)).
 			put(optionsKey, config.Data(md.Options)).
-			put(redemptionReservesKey, uint64Bytes(md.RedemptionReserves)).
-			put(refundReservesKey, uint64Bytes(md.RefundReserves)).
```

Comment:
Possible simplification to extract from this PR. (`UpdateOrderMetaData` should not update these constant values that are supposed to be stored on creation in `UpdateOrder`)